include/SyntaxNode.h

```diff
@@ -407,6 +411,12 @@ enum class SyntaxKind : uint16_t {
     LetPort,
     LetPortList,
     LetDeclaration,
+    AssertionVariableDeclaration,
```

Comment:
Remember to remove the unused SyntaxKinds. I believe there are warnings in the build about it.